lib/carbon/tests/test_storage.py

```diff
@@ -0,0 +1,107 @@
+import os
+from unittest import TestCase
+from mock import patch
+
+
+# class NoConfigSchemaLoadingTest(TestCase):
+
+#     def setUp(self):
+#         settings = {
+#             'CONF_DIR': '',
+#         }
+#         self._settings_patch = patch.dict('carbon.conf.settings', settings)
+#         self._settings_patch.start()
+
+#     def tearDown(self):
+#         self._settings_patch.stop()
+
+#     def test_loadAggregationSchemas_load_default_schema(self):
+#         from carbon.storage import loadAggregationSchemas, defaultAggregation
+#         schema_list = loadAggregationSchemas()
+#         self.assertEquals(len(schema_list), 1)
+#         schema = schema_list[0]
+#         self.assertEquals(schema, defaultAggregation)
+
+#     def test_loadStorageSchemas_raise_CarbonConfigException(self):
+#         from carbon.storage import loadStorageSchemas
+#         from carbon.exceptions import CarbonConfigException
+#         with self.assertRaises(CarbonConfigException):
+#             loadStorageSchemas()
```

Comment:
As I explained in the PR, I was hoping someone might be able to enlighten me regarding this mock persistence across tests. If not, sure I’ll remove it.